tests/run/test_asyncgen.py

```diff
@@ -974,7 +974,7 @@ async def gen():
                 DONE += 1000
                 yield v * 2
             finally:
-                assert sys.exc_info()[0] == ZeroDivisionError
+                self.assertEqual(sys.exc_info()[0], ZeroDivisionError)
```

Comment:
This file is a copy from CPython. It should not be changed syntactically in order to ease future updates.
```suggestion
                assert sys.exc_info()[0] == ZeroDivisionError
```

Reply:
Ok, seems reasonable